Utils/hdinsight-node-common/src/com/microsoft/azure/sqlbigdata/sdk/cluster/SqlBigDataLivyLinkClusterDetail.java

```diff
@@ -110,8 +113,11 @@ public SparkSubmitStorageTypeOptionsForCluster getStorageOptionsType() {
     @Override
     @NotNull
     public String mapInternalUrlToPublic(@NotNull String url) {
-        // FIXME: Hardcode for only output Yarn NM connection URL
-        // We should extract application ID from internal URL and map it to output
-        return getYarnNMConnectionUrl();
+        // Extract application ID from internal URL
+        // url example: http://mssql-master-pool-0.service-master-pool:8088/proxy/application_1544743878531_0035/
```

Comment:
Does the `mssql-master-pool-0.service-master-pool:8088` has relationship with yarn service?